angel_system/impls/detect_activities/pytorchvideo_slow_fast_r50.py

```diff
@@ -81,7 +81,7 @@ def get_model(self) -> "torch.nn.Module":
             model_device = torch.device('cpu')
             if self._use_cuda:
                 if torch.cuda.is_available():
-                    model_device = torch.device(device=self.cuda_device)
+                    model_device = torch.device(device=self._cuda_device)
```

Comment:
Good catch, I did see this earlier and decided to ignore it for the moment because it wasn't the swinb algo.